core/templates/pages/blog-dashboard-page/modal-templates/upload-blog-post-thumbnail.component.ts

```diff
@@ -94,7 +97,7 @@ export class UploadBlogPostThumbnailComponent implements OnInit {
   }
 
   reset(): void {
-    this.uploadedImage = null;
+    this.uploadedImage = false;
```

Comment:
Wait why is this set to `false` when the type is `SafeResourceUrl`?

Reply:
Previous contributor use `false`, which is bad typed. Use null everywhere now.
And `SafeResourceUrl` is child of `safeValue` means we can use any safe type here. Example: string, number, boolean etc.